CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Why this change is needed?

Reply:
Sorry; my bad! I thought CMake <=3.20 would complain with `SET CMP0116 OLD`. I'll change it back to 3.18.